hawkular-inventory-impl-cassandra/src/main/java/org/hawkular/inventory/impl/cassandra/CassandraBackend.java

```diff
@@ -195,8 +257,15 @@ public boolean hasRelationship(Row entity, Relationships.Direction direction, St
 
     @Override
     public Set<Row> getRelationships(Row entity, Relationships.Direction direction, String... names) {
```

Comment:
I know that the model changes a lot, but could you update this reflect the latest model, where relationships are modelled in 3 tables? One "main" table, and in/out tables that are meant used for hops and filtering.